usr/share/rear/restore/NETFS/default/51_set_capabilities.sh

```diff
@@ -0,0 +1,11 @@
+# restore capabilities if capabilities are in the backup
+	if test -s $VAR_DIR/recovery/capabilities ; then
+		Log "Restoring Capabilities."
+		if which setcap >/dev/null 2>&1 ; then
+			while read file cap ; do
+				setcap $cap ${TARGET_FS_ROOT}/${file}	
+			done < <(cat $VAR_DIR/recovery/capabilities | sed 's/=//')
+		else
+			Log "setcap binary not found."
```

Comment:
In my opinion the `REQUIRED_PROGS` in `prep/default/95_check_missing_programs.sh` is not working correctly. Bash is not my favorite thing so maybe I am wrong.  
Due to `MISSING_PROGS=("")` the later `test -n "$MISSING_PROGS"` always detects the first element as null. 
```
[root@cen2 rear]# MISSING_PROGS=("")
[root@cen2 rear]# MISSING_PROGS=( "${MISSING_PROGS[@]}" "missing_bin" )
[root@cen2 rear]# if test -n "$MISSING_PROGS" ; then echo "something is missing" ;fi
```

What works for me is 
```
if [ ${#MISSING_PROGS[@]} -ne 1 ] ; then echo "something is missing" ;fi
```